plugin/pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -97,6 +97,7 @@ func (p *podAffinityPriorityMap) processTerm(term *api.PodAffinityTerm, podDefin
 				if p.failureDomains.NodesHaveSameTopologyKey(node, fixedNode, term.TopologyKey) {
 					p.counts[node.Name] += weight
 				}
+				// TODO i think we need to  add processing logic for pod labels here?
```

Comment:
>
I think ProcessTerm should be, by definition, involvving the pod matches term namespace / selector information contents in its calculation.

Yes - and it involes. This is delegated PodMatchesTermsNamespaceAndSelector which is called ~10 lines above __as part of processTerm__ function.